deepcell_tracking/utils.py

```diff
@@ -330,3 +339,312 @@ def trks_stats(filename):
     print('Total number of divisions                  - ', total_divisions)
     print('Average cell density (cells/100 sq pixels) - ', avg_cells_per_sq_pixel * 100)
     print('Average number of frames per track         - ', int(avg_num_frames_per_track))
+
+
+def get_max_cells(y):
+    """Helper function for finding the maximum number of cells in a frame of a movie, across
+    all frames of the movie. Can be used for batches/tracks interchangeably with frames/cells.
+
+    Args:
+        y (np.array): Annotated image data
+
+    Returns:
+        max_cells (int): The maximum number of cells in any frame
+    """
+    max_cells = 0
+    for frame in range(y.shape[0]):
+        cells = np.unique(y[frame])
+        n_cells = cells[cells != 0].shape[0]
+        if n_cells > max_cells:
+            max_cells = n_cells
+    return max_cells
+
+
+def normalize_adj_matrix(adj, epsilon=1e-5):
```

Comment:
Please include a docstring as well as tests for this function for some basic accuracy stuff. I could easily see this function having a silent error.